### PR TITLE
(DOCSP-23805) [BAAS] Raise Request Timeout from 120s -> 150s

### PR DESCRIPTION
**Pull request by nlarew**
### Jira

- (DOCSP-23805) [BAAS] Raise Request Timeout from 120s -> 150s

### Staged Changes (Requires MongoDB Corp SSO)

- [Functions > Constraints](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/request-timeout/functions/#constraints)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### CODE REVIEW

**Review by mongomoe — APPROVED** [review]
LGTM

File: snooty.toml [hunk 1]
```diff
@@ -79,6 +79,7 @@ max-graphql-resolvers = "ten"
 realm = "realm"
 realms = "realms"
 realm-web = "Realm Web SDK"
+request-timeout-time = "120 seconds"
```
Comment (mongomoe) [inline snooty.toml:82]:
```suggestion
request-timeout-time = "150 seconds"
```
Looks like you forgot to actually change the timeout to 150

**Review by mongomoe — COMMENTED** [review]
1 Needed fix actually, then feel free to merge it

File: snooty.toml (hunk 1 above)
Comment (nlarew) [reply snooty.toml:82]: 😓

**Pull request merged**